src/features/meetings/monthly_view/index.tsx

```
@@ -0,0 +1,1157 @@
+              return showDoublePerson[index] ? (
+                <PersonSelector
+                  key={`aux-classroom-counselor-${index}`}
+                  week={value}
+                  notShowIcon
+                  label={t('tr_chairman')}
+                  type={AssignmentCode.MM_AuxiliaryCounselor}
+                  assignment="MM_Chairman_B"
+                  readOnly={false}
+                />
+              ) : (
+                <Box flex={1}></Box>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)